libs/core/src/components.d.ts

```diff
@@ -149,11 +149,53 @@ export namespace Components {
          */
         "variant": 'inline' | 'plain';
     }
+    interface SageTab {
+        "index": number;
+        "parentComponentId": string;
+        "selected": boolean;
+        /**
+          * Sets the related tab name, this name must match a `sage-tabpanel`'s tab property
+         */
+        "tab": string;
+        "variant": string;
+    }
+    interface SageTabpanel {
+        "parentComponentId": string;
+        "selected": boolean;
+        /**
+          * Sets the related tab name, this name must match a `sage-tab`'s tab property
+         */
+        "tab": string;
+        "variant": string;
+    }
+    interface SageTabs {
+        /**
+          * Sets starting active tab and maintains active tab as component re-renders
```

Comment:
Since this is a string, I would change the description a bit.

`Sets the starting active tab name and maintains the name as the component re-renders`